### PR TITLE
New importer for VCTK corpus

### PR DESCRIPTION
**Pull request by rhamnett**
VCTK corpus contains high quality samples (48khz) of english transcription c.45 hours.

Importer converts samples to 16khz and creates suitable csv files manually split into train/dev/test after random shuffle (not sure if this is correct). A few bits of missing data and erroneous records are stripped.

VCTK Corpus - https://datashare.is.ed.ac.uk/handle/10283/2651
Used in Wavenet paper - https://arxiv.org/pdf/1609.03499.pdf

### CODE REVIEW

**Comment by rhamnett** [comment]
https://homepages.inf.ed.ac.uk/jyamagis/page3/page58/page58.html

COPYING
This corpus is licensed under Open Data Commons Attribution License (ODC-By) v1.0.
http://opendatacommons.org/licenses/by/1.0/
http://opendatacommons.org/licenses/by/summary/

File: bin/import_vctk.py [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+        csv_line = f"{file},{file_size},{utterence_clean}\n"
+        csv.append(csv_line)
+
+    random.shuffle(csv)
+
+    train_data = csv[:37000]
+    dev_data = csv[37000:40200]
+    test_data = csv[40200:]
+
+    print(len(train_data))
```
Comment (lissyx) [inline bin/import_vctk.py]: Is this for checking purpose? Maybe `assert` here instead, if that's the case.

File: bin/import_vctk.py [hunk 2]
```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+
+# VCTK used in wavenet paper https://arxiv.org/pdf/1609.03499.pdf
```
Comment (lissyx) [inline bin/import_vctk.py]: Can you mention the license as well?

**Comment by rhamnett** [comment]
Closing PR will submit again with one clean commit

**Pull request closed without merging**